src/config/config_test.go

```diff
@@ -0,0 +1,94 @@
+package config
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/models"
+)
+
+func TestConfig(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "config_test")
+}
+
+var _ = Describe("Config tests", func() {
+	BeforeEach(func() {
+
+	})
+
+	Context("Fetching installer args", func() {
```

Comment:
A few things here.
`BeforeEach` shouldn't be present if it's empty.

While `Describe` and `Context` function identically they have slightly different semantics.
Typically `Describe` would be used with a specific function or method name. In BDD testing frameworks the idea is that you're _describing_ how the object or method should function.

A context would be used with shared setup code to define the _context_ for the individual examples (tests). So if you wanted to get really pedantic you could do something like:

```go
var _ = Describe("SetInstallerArgs", func() {
  var args string
  Context("with valid args", func() {
    BeforeEach(func() {
      args = "[\"arg1=foo\", \"arg2=bar\"]"
    })
    It("sets the value correctly", func() {
      ...
    })
  })
})
var _ = Describe("SetDefaults", func() {
...
```

The idea is to have the whole thing read like a sentence describing what kind of behavior the test is expecting so if this test fails you would see `SetInstallerArgs with valid args sets the value correctly`. You then immediately know what case isn't working for what function.

You don't have to go quite that far in this PR, I generally will put the setup code in the example when there would only be one case in a context, but I do think using top-level `Describe` blocks for the different functions and getting rid of the context and empty `BeforeEach` block would be an improvement.